isic/ingest/templates/ingest/cohort_files.html

```diff
@@ -44,7 +44,7 @@ <h2>Zip Files</h2>
                 {% comment %}
                   TODO: put in a status for rejected zips
                   {% endcomment %}
-                {% for zip in cohort.zips.all %}
+                {% for zip in cohort.zip_uploads.all %}
```

Comment:
Maybe change the iterator value here too, to make it easier to grep for in the future?